src/pages/authentication-pages/signup/Signup.jsx

```diff
@@ -0,0 +1,85 @@
+function Signup() {
+  return (
+    <section class="login_form_container d-flex">
+      <form
+        class="card-basic login_form"
+        onsubmit="event.preventDefault(); checkFormValidation()"
```

Comment:
I am not able to find checkFormValidation method here. Please move event.preventDefault() inside that method do not call it like this.

Reply:
done